docs/Activity.md

```diff
@@ -0,0 +1,385 @@
+# `interface Activity`
+
+`Activity` is a Microsoft Bot Framework representation of actions made by
+conversation participants.
+
+The full specification of the `Activity` interface can be found in the
+Microsoft Bot Framework SDK documentation at https://docs.microsoft.com/en-us/azure/bot-service/index-bf-sdk
+and in the Microsoft Bot Framework SDK specs at https://github.com/microsoft/botframework-sdk/tree/main/specs.
+
+However, at the moment, Tyntec WhatsApp Adapter supports only a limited subset
+of activities. This means, that there are stricter requirements for activities
+passed to the adapter as well as for activities created by the adapter.
+
+The only supported activity type is a [WhatsApp message](#whatsapp-message-activity)
+(`type: "message", channelId: "whatsapp"`).
+
+
+## WhatsApp Message Activity
+
+The sender of the activity (`from`) must be defined. The channel account ID
```

Comment:
I think list generally make us write less words and thus prevent big chunks of text (which are, sadly, discouraging for the common reader). So, in general, i think they are better in terms of readability.

However, the emphasis with (MUST MUST NOT MAY) works on similar level (I expect the reader to scan these parts rather than read thoroughly (yes, it is reference, but you mostly care about the key pair (from and MUST, deliveryMode and MUST NOT)). And if this mimics the way MS is doing documentation, it can seem more familiar to the active user, which can be better call here.

So, I would personally go with bullet lists here because:
- They improve readability (especially if we are giving bunch of "key-pairs") 
- They are easier to maintain
- They give less space to confusion (but also have less space to explain something)

Reply:
What about this? https://github.com/yet-another-radek/botbuilder-adapter-tyntec-whatsapp/compare/2025655..f66608b